test/app-tap/gh-2717-no-quit-sigint.test.lua

```diff
@@ -19,7 +19,7 @@ local TARANTOOL_PATH = arg[-1]
 local test = tap.test('gh-2717-no-quit-sigint')
 
 test:plan(8)
-local cmd = 'INPUTRC=Pheiphe2 ' .. TARANTOOL_PATH .. ' -i 2>&1'
```

Comment:
Why is `Pheiphe2` any better than `non_existent_file`? Why isn't another test updated? IMO better create a real inputrc file for tests.

Anyway, the two patches should be squashed.

Reply:
Sorry - thought you replaced `non_existent_file` with `Pheiphei2`. Never mind - everything's fine. Still, I wouldn't split patches in such a fine grained way - "While we are at it, fix the non existent file name" in the commit message would do just fine.